### DISCUSSION TITLE
invalid ignore file should be reported

### DISCUSSION THREAD

**Discussion by itaysk** [aquasecurity/trivy#7596] [Ideas]
### Description

Not a critical issue but when using `--ignorefile badpath.yaml` with an invalid path, Trivy silently ignores the error. I think it should result in an error or a warning. For reference, the simlar flag `--ignore-policy` will not just error but also stop trivy.

### Target

None

### Scanner

None

**Comment by simar7** [comment]
Yes, I can't seem to find the previous discussion that we had in general regarding Trivy's behavior when supplied user input isn't valid/missing. But I think in general, we should warn or inform the user about such a situation so that the scan results or Trivy's behavior isn't questioned.

**Comment by simar7** [comment]
We already have an issue to track this https://github.com/aquasecurity/trivy/issues/7093
